From aec50d94cb6b02ebff5c29133c785b7241aec98f Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Tue, 11 Mar 2025 15:14:21 +0100
Subject: [PATCH] ARM: configs: stm32mp2: enable IKCONFIG and /proc/config.gz
 for aarch32

IKCONFIG is used during system tests to check the Linux kernel
configuration.
It provides documentation of which kernel options are used in the
running kernel, using /proc/config.gz.

Change-Id: I08fc2451d867361b874fcf2d9846aec2fbb14b81
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/443465
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm/configs/mp2-fragment-04-multiv7-addons.config | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm/configs/mp2-fragment-04-multiv7-addons.config
+++ b/arch/arm/configs/mp2-fragment-04-multiv7-addons.config
@@ -1,3 +1,4 @@
 CONFIG_ARM_LPAE=y
 CONFIG_ARM_SMC_WATCHDOG=y
-
+CONFIG_IKCONFIG=y
+CONFIG_IKCONFIG_PROC=y
